packages/client/src/v2-events/features/workqueues/EventOverview/components/EventHistory/EventHistoryDialog/EventHistoryDialog.tsx

```diff
@@ -24,7 +24,7 @@ import { getActionTypeSpecificContent } from './actionTypeSpecificContent'
 export const eventHistoryStatusMessage = {
   id: `v2.events.history.status`,
   defaultMessage:
-    '{status, select, CREATE {Draft} NOTIFY {Sent incomplete} VALIDATE {Validated} DRAFT {Draft} DECLARE {Sent for review} REGISTER {Registered} PRINT_CERTIFICATE {Certified} REJECT {Rejected} ARCHIVE {Archived} MARKED_AS_DUPLICATE {Marked as a duplicate} READ {Viewed} ASSIGN {Assigned} UNASSIGN {Unassigned} other {Unknown}}'
+    '{status, select, CREATE {Draft} NOTIFY {Notified} VALIDATE {Validated} DRAFT {Draft} DECLARE {Sent for review} REGISTER {Registered} PRINT_CERTIFICATE {Certified} REJECT {Rejected} ARCHIVE {Archived} MARKED_AS_DUPLICATE {Marked as a duplicate} READ {Viewed} ASSIGN {Assigned} UNASSIGN {Unassigned} other {Unknown}}'
```

Comment:
Just confirming that this is how we want it? I previously changed it from Notified -> Sent incomplete as specified eg. here: https://opencrvsworkspace.slack.com/archives/C07NQAULENB/p1752072290118529